src/components/pages/pages-test.stories.tsx

```diff
@@ -121,3 +123,90 @@ export const DefaultStory = ({ initialPageIndex }: PageStoryProps) => {
 
 DefaultStory.storyName = "default";
 DefaultStory.parameters = { args: { initialPageIndex: 0 } };
+
+export const PagesComponent = ({ ...props }: PageStoryProps) => {
+  const [isOpen, setIsOpen] = useState(false);
```

Comment:
Why is this not `React.useState(false)`?

Reply:
because `useState` is directly imported from `React` at the top of this file